coralproject/talk
# [CORL-1502] Publish Static Files on Tag

## What does this PR do?

When a release is tagged, the static files are published to the GitHub release.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



Wait for CircleCI to run, and watch during the next release to see that it tags it!
